spec/requests/purchases_requests_spec.rb

```diff
@@ -48,11 +49,13 @@
 
     describe "GET #new" do
       subject do
+        organization.update!(default_storage_location: storage_location)
         get new_purchase_path
         response
       end
 
       it { is_expected.to be_successful }
+      it { is_expected.to include(storage_location.name) }
     end
```

Comment:
Currently, the second expectation fails with [this traceback](https://gist.github.com/lenikadali/5c3ee9205d316c5fd3d1eb3a10386a77) even though when we do Ctrl+F, we are able to find it in the traceback:

```
 `       +     "        <div class=\"form-group select required purchase_storage_location\"><label class=\"select required\" for=\"purchase_storage_location_id\">Storage Location <abbr title=\"required\">*</abbr></label><div class=\"input-group\"><select class=\"form-control select required\" name=\"purchase[storage_location_id]\" id=\"purchase_storage_location_id\"><option value=\"1\">Pawane Location</option></select></div></div>\n" +
```

Reply:
This is probably because this magic method translates into this:

```
it 'should include storage location name' do
  expect(response).to include(storage_location.name)
end
```

In this case you likely want to check `response.body`, not `response`. Probably better to use the extended format rather than the succinct one.

Reply:
Thanks a lot for the pointer 🙌🏿 

Fixed in [fbab592](https://github.com/rubyforgood/human-essentials/pull/4722/commits/fbab592abe2ca370879214a9fc9681332f8b8e5d)